api/spec/src/paths/tags/post.yaml

```diff
@@ -0,0 +1,14 @@
+description: 'Create new tag'
+consumes:
+- application/json
+produces:
+- application/json
+- in: body
+  name: body
+  description: 'Tag object that needs to be added'
+  required: true
+  schema:
+    $ref: file://src/open_api.yaml#/definitions/TagDefault
+responses:
+  405:
```

Comment:
We need to cover at least several other cases:  
- tag already exists in database
- tag is blacklisted (for example profane language)
- tag is too long  

We could either:  
- add another generic response handling other cases and returning specific error message, which would depend on the situation 
- trying to fit each of those cases into specific response code, along with error message